tests/unittests/BackendCorrectnessTest.cpp

```diff
@@ -58,6 +58,9 @@ TEST_P(BackendCorrectnessTest, convTest) {
 
 TEST_P(BackendCorrectnessTest, extract3Dtest) {
   CHECK_IF_ENABLED();
+#ifndef GLOW_WITH_CPU
```

Comment:
What's the reason for this? CHECK_IF_ENABLED should handle this

Reply:
it seems that the test actually is broken.

> inferExtract3D(&inputs, &out1, "CPU");
 inferExtract3D(&inputs, &out2, "Interpreter");

needs to be
inferExtract3D(&inputs, &out1, backendName_);
inferExtract3D(&inputs, &out2, "Interpreter");

Reply:
Done!

Reply:
@pjaaskel Could you have a look at how `glow::backendTestBlacklist` is used in Glow. It is described in docs and it is also used by e.g. Habana backend. It would be much cleaner to just blacklist any tests that don't need to be executed by the OpenCL backend. This way there is no need for any preprocessor-based conditional compilation.

Reply:
@pjaaskel Ping! Do you still plan to land this PR?

Reply:
Yes, it's in my task list. Hopefully I'll get into it soon.